nano/lib/ipc_client.cpp

```diff
@@ -107,6 +107,8 @@ class socket_client : public nano::ipc::socket_base, public channel, public std:
 		}));
 	}
 
+	// TODO: investigate clang-tidy warning about recursive call chain
```

Comment:
I wouldn't recommend adding a TODO comment for something that comes from outside the source code (a tool), since the warning will eventually come up when running clang-tidy.

Reply:
Yeah that's more a note for myself to be honest, I intend to investigate and fix those two very soon so it's easier to see the todo rather than look up the warning again. But I agree with your advice generally.